Source/ASNetworkImageNode.mm

```diff
@@ -581,21 +619,25 @@ - (void)_lazilyLoadImageIfNecessary
             }
 
             if (animatedImage != nil) {
+              ASDN::MutexUnlocker u(__instanceLock__);
               self.animatedImage = animatedImage;
```

Comment:
I hate to say it but I think we should have a `_locked_` version of this too instead of unlocking…